ft232/libftdi.py

```diff
@@ -421,3 +421,6 @@ def setBitMode(self, mask, mode=BM_RESET):
 
     def gpio(self):
         return GPIO(self)
+
+    def spi(self, Clock=1000000, Mode=0, BitOrder=SPI.BIT_ORDER_MSB):
```

Comment:
Similar issue with the GPIO class but I'd probably suggest a different solution. Create a class that inherits from Ft232 and layers on top the SPI specific functions. This makes sense because the user should know when they are writing the code whether they are dealing with SPI or UART hardware and should pick the appropriate class.